.github/workflows/needsinfohelper.yaml

```diff
@@ -10,5 +10,4 @@ jobs:
         if: ${{ ! startsWith(github.event.comment.body, 'This issue has been marked as "needs info" 4 weeks ago.') && ! startsWith(github.event.comment.body, 'This bug report did not receive an update in the last 4 weeks.')}}
         with:
           labels: 'needs info'
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          github_token: ${{ secrets.github_token }}
```

Comment:
I am not sure if this makes a difference, as e.g. here it is also capital:
https://github.com/nextcloud/desktop/blob/master/.github/workflows/rebase.yml#L18

Reply:
I checked the log and I am under the impression I should have used the upper case name for the github token

thanks for the review